src/ai/backend/agent/exception.py

```diff
@@ -34,6 +34,12 @@ class UnsupportedBaseDistroError(RuntimeError):
     pass
 
 
+class ContainerCreationError(Exception):
+    def __init__(self, *args, container_id, **kwargs):
```

Comment:
`*args` should appear before positional argument `container_id`.
```suggestion
    def __init__(self, container_id, *args, **kwargs):
```

Reply:
I locate `container_id` after positional argument cause I intended to set `container_id` only by keyword argument. is it okay?

Reply:
I updated it to `def __init__(self, container_id, *args, **kwargs):`